packages/vscode-extension/src/commands/FolderType.ts

```diff
@@ -0,0 +1,5 @@
+export interface FolderType {
+	name: string | undefined;
```

Comment:
Why do we need to specify undefined here? Can we use optional here instead "?"?

Reply:
I have deleted it and it´s working fine